packages/core/src/components/Tabs/Tab.tsx

```diff
@@ -163,11 +187,40 @@ export default withStyles(({ color, font, pattern, unit, ui, transition }) => ({
     borderTopRightRadius: ui.borderRadius,
   },
 
-  tabButton_small: {
-    ...font.textSmall,
+  tabButton_secondary: {
+    ...pattern.regularButton,
+    paddingTop: unit / 2,
+    paddingBottom: (unit / 8) * 5,
+    fontWeight: 'normal',
+    justifyContent: 'center',
+    border: `${ui.borderWidth}px solid ${color.clear}`,
+    backgroundColor: color.clear,
+    borderRadius: ui.borderRadius,
+    ':hover': {
+      borderColor: color.accent.borderHover,
+      backgroundColor: color.accent.bgHover,
+    },
   },
 
-  tabButton_active: {
+  tabButton_selected: {
     color: color.accent.textActive,
   },
+
+  tabButton_secondary_selected: {
+    borderColor: color.accent.borderActive,
+    backgroundColor: color.accent.bg,
+    ':hover': {
+      borderColor: color.accent.borderActive,
+      backgroundColor: color.accent.bg,
```

Comment:
why is the hover state the same? use `color.accent.bgHover` here.

Reply:
@stefhatcher this is the desired design cc @adamirl

Reply:
chatting it over with @adamirl . buttons in this nav should follow similar background color hover state patterns as other buttons. ToggleButtonController is similar and has background hover states http://airbnb.io/lunar/?path=/story/core-togglebuttoncontroller--a-list-of-single-select-toggle-buttons

Reply:
e.g. selected's hover state is `color.accent.bgHover` and the other non-selected hover states should be one darker: `color.core.neutral[1]` (probably border too)

Reply:
@stefhatcher like this? ![Screen Recording 2019-10-09 at 3 54 14 PM](https://user-images.githubusercontent.com/205004/66526486-b0534600-eaad-11e9-8556-fd858c5a250c.gif)

Reply:
done